recipe/meta.yaml

```diff
@@ -85,11 +131,11 @@ package:
 
 source:
   - path: ./build.py
-  - url: https://{{ subdomain }}.nvidia.com/compute/cuda/{{ download_dir }}/local_installers/cuda_{{ version }}_{{ version_patch }}_linux.run  # [linux64]
+  - url: https://{{ subdomain }}.nvidia.com/compute/cuda/{{ download_dir }}/local_installers/cuda_{{ version }}_{{ version_patch }}_linux{{ blob_ext }}  # [linux64]
     md5: {{ cudavars[major_minor]["checksums"]["linux64"] }}  # [linux64]
-  - url: https://{{ subdomain }}.nvidia.com/compute/cuda/{{ download_dir }}/local_installers/cuda_{{ version }}_{{ version_patch }}_linux_ppc64le.run  # [ppc64le]
+  - url: https://{{ subdomain }}.nvidia.com/compute/cuda/{{ download_dir }}/local_installers/cuda_{{ version }}_{{ version_patch }}_linux_ppc64le{{ blob_ext }}  # [ppc64le]
     md5: {{ cudavars[major_minor]["checksums"]["ppc64le"] }}  # [ppc64le]
-  - url: https://{{ subdomain }}.nvidia.com/compute/cuda/{{ download_dir }}/local_installers/cuda_{{ version }}_{{ version_patch }}_win10.exe  # [win]
+  - url: https://{{ subdomain }}.nvidia.com/compute/cuda/{{ download_dir }}/local_installers/cuda_{{ version }}_{{ version_patch }}_win10{{ blob_ext }}  # [win]
     md5: {{ cudavars[major_minor]["checksums"]["win"] }}  # [win]
```

Comment:
FWIW we could use `fn` to specify the filename on download. This could help us standardize the extension type (and any other parameters). Just a thought :)

Reply:
Yeah, in 9.2 this gets even more annoying though, because windows doesn't have a `version_patch` value. I can change if you want though to have consistent filenames

Reply:
Yeah happy to leave as-is. Just thinking about how to keep complexity to a minimum. Happy to defer to your judgement :)

Reply:
Yeah, thanks for pushing me here. I have streamlined this now in a way that will really help for 9.2 too.

Reply:
Thanks Anthony! Looks good 😄